operator/v1/0000_30_metal3provisioning_00_config.crd.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: metal3provisioning.operator.openshift.io
+spec:
+  group: operator.openshift.io
+  names:
+    kind: Metal3Provisioning
+    listKind: Metal3ProvisioningList
+    singular: metal3provisioning
+  scope: ""
+  preserveUnknownFields: false
+  subresources:
+    scale:
+      labelSelectorPath: .status.selector
+      specReplicasPath: .spec.replicas
+      statusReplicasPath: .status.availableReplicas
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: "Metal3Provisioning contains configuration used by the Provisioning
+        service (Ironic) to provision baremetal hosts. \n Metal3Provisioning is created
```

Comment:
Just curious, do we actually need the \n's here?  None of the other longer texts have them..